### PR TITLE
Prepare crates.io release

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
+categories = ["no-std", "hardware-support", "embedded"]
+license = "BSD-3-Clause"
+repository = "https://github.com/Sensirion/sensirion-i2c-rs"
 edition = "2018"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-embedded-hal = "1.0.0-alpha.1"
+embedded-hal = "*"
```
Comment (dbrgn) [inline Cargo.toml:16]: This won't be allowed when publishing to Crates.io... You have to specify a version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Changelog
+
+This project follows [semantic versioning](https://semver.org/).
+
+## [0.1.0] (2020-08-20)
+
+ * Initial version which implements the CRC-8 algorithm commonly used by
+   Sensirions sensors and a few I2C helper functions.
```
Comment (abrauchli) [inline CHANGELOG.md:8]:
```suggestion
   Sensirion's sensors and a few I2C helper functions.
```

**Review by abrauchli — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (rnestler) [reply Cargo.toml:16]: Thanks, good catch :slightly_smiling_face:

Comment (rnestler) [reply Cargo.toml]: Fixed.

Comment (rnestler) [reply Cargo.toml]: (My usual approach to add the most recent version of a crate is to first just specifiy `*` in `Cargo.toml`, then check `Cargo.lock` which version that is and update `Cargo.toml` with that version. I should probably rethink that approach :wink:)

Comment (dbrgn) [reply Cargo.toml]: Mine is opening `docs.rs/<crate-name>` :smile:

**Pull request merged**